db-scripts/src/main/resources/migration.sql

```diff
@@ -599,3 +590,24 @@ CREATE TABLE `data_access_tokens` (
     FOREIGN KEY (`USERNAME`) REFERENCES `users` (`EMAIL`) ON DELETE CASCADE
 );
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.9.0";
+
+##version: 2.10.0
+INSERT INTO reference_genome_gene (ENTREZ_GENE_ID, CYTOBAND, EXONIC_LENGTH, CHR, REFERENCE_GENOME_ID)
+SELECT
+    ENTREZ_GENE_ID,
+    CYTOBAND,
+    LENGTH,
+    SUBSTRING_INDEX(SUBSTRING_INDEX(SUBSTRING_INDEX(gene.CYTOBAND,IF(LOCATE('p', gene.CYTOBAND), 'p', 'q'), 1),'q',1),'cen',1),
+    1
+FROM `gene`
+WHERE NOT EXISTS (SELECT * FROM reference_genome_gene);
+ALTER TABLE `gene` DROP COLUMN `CYTOBAND`, DROP COLUMN `LENGTH`;
+ALTER TABLE `cancer_study` ADD COLUMN `REFERENCE_GENOME_ID` INT(4) DEFAULT 1,
+                           ADD CONSTRAINT `FK_REFERENCE_GENOME` FOREIGN KEY (`REFERENCE_GENOME_ID`)
+                               REFERENCES `reference_genome`(`REFERENCE_GENOME_ID`) ON DELETE CASCADE;
```

Comment:
in `cgds.sql` this is `ON DELETE RESTRICT` and here it is `ON DELETE CASCADE`. Please adjust to make them the same. My preference goes to `RESTRICT` in this case. 
As a general comment: please put the cgds.sql and migration.sql side by side and check very carefully to make sure both scripts end up with the same result.

Reply:
what do you think @sheridancbio , should we go with RESTRICT? I vote for ON DELETE CASCADE.

Reply:
any updates here?

Reply:
what do you think, @sheridancbio? We may never want to remove any reference genomes. Either way really does not matter I would say.

Reply:
@pieterlukasse , changed to **_ON DELETE RESTRICT_** to be consist with cgds.sql.